packages/react-composites/src/composites/CallComposite/CallControls.tsx

```diff
@@ -37,30 +70,40 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
 
   return (
     <ControlBar layout="dockedBottom">
-      <CameraButton data-ui-id="call-composite-camera-button" {...cameraButtonProps} showLabel={!compressedMode} />
-      <MicrophoneButton
-        data-ui-id="call-composite-microphone-button"
-        {...microphoneButtonProps}
-        showLabel={!compressedMode}
-      />
-      <ScreenShareButton {...screenShareButtonProps} showLabel={!compressedMode} />
-      {props.showParticipantsControl && (
+      {hiddenElements?.cameraButton !== true && (
```

Comment:
why not just !hiddenElements?.cameraButton instead?

Reply:
Readability. `!h` is easy to miss.
I missed it myself a couple of times.

Reply:
And also because it takes some mental overhead to quickly understand double negatives .